lib/components/narrative/default/default-itinerary.js

```diff
@@ -48,18 +90,20 @@ const ITINERARY_ATTRIBUTES = [
         else return formatTime(itinerary.startTime, options)
       }
       return (
-        <span>
-          {formatTime(itinerary.startTime, options)}
-          —
-          {formatTime(itinerary.endTime, options)}
-        </span>
+        formatTime(itinerary.startTime, itinerary.endTime, options.timeFormat)
       )
     }
   },
   {
     id: 'cost',
     order: 2,
-    render: (itinerary, options) => getTotalFareAsString(itinerary)
+    render: (itinerary, options) => {
+      // Get unformatted transit fare portion only (in cents).
+      const { transitFare } = calculateFares(itinerary)
+      return (
+        <FormattedNumber value={transitFare / 100} style='currency' currencyDisplay='narrowSymbol' currency='USD' />
```

Comment:
The currency should be dynamically populated, perhaps with a fallback to USD or maybe even some configurable value. I think if OTP is configured properly, it will output the currency code as part of an itinerary response.

Reply:
Was updated in https://github.com/opentripplanner/otp-react-redux/pull/375/commits/d0f962fcf847a7ba0d3b9a9b9fa33501ce41e00a.